packages/@tailwindcss-node/src/compile.ts

```diff
@@ -74,6 +74,8 @@ async function loadStylesheet(id: string, base: string, onDependency: (path: str
   let resolvedPath = await resolveCssId(id, base)
   if (!resolvedPath) throw new Error(`Could not resolve '${id}' from '${base}'`)
 
+  onDependency(resolvedPath)
```

Comment:
Oh no, I was so certain I did add this. Thank you for fixing this 💅

Reply:
No worries! This made a very easy fix for me. Spend 10x more time on the integration tests 😂